widgetssdk/src/main/java/com/glia/widgets/filepreview/domain/usecase/IsFileReadyForPreviewUseCase.kt

```diff
@@ -0,0 +1,11 @@
+package com.glia.widgets.filepreview.domain.usecase
+
+import com.glia.androidsdk.chat.AttachmentFile
```

Comment:
can you please make this class internal

Reply:
fixed 45537a8b2ebe0acb61caa153d8e805ed7723169b